app/controllers/users/backup_code_setup_controller.rb

```diff
@@ -10,10 +10,6 @@ class BackupCodeSetupController < ApplicationController
     before_action :set_backup_code_setup_presenter
     before_action :apply_secure_headers_override
 
-    def index
-      @presenter = BackupCodeCreatePresenter.new
-    end
```

Comment:
Personally I like keeping the empty actions so we remember they're there? But not a big deal
```ruby
def index
end
```

Reply:
same!

Reply:
Sure. In retrospect, I agree. There's also stub below for `edit` and `confirm_delete`, presumably for the same purpose.

Restored in a15aa9b68.